src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsMappingBase.java

```diff
@@ -10,8 +13,10 @@ public class HearingsMappingBase {
 
     public static final String CASE_CREATED = "2022-04-01";
 
-    protected HearingsMappingBase() {
+    public final ReferenceData referenceData;
 
+    protected HearingsMappingBase() {
+        referenceData = new ReferenceData(new HearingDurationsService(), new SessionCategoryMapService());
```

Comment:
Ideally when it comes to unit tests you want to mock any external dependencies to make sure you're only testing the current 'unit'. Using the full external dependency introduces a reliance on what it's current implementation - which could very well change. A change there would probably break your tests. You want to mock out the external dependency, using representative mock data for you tests. That way you have full control over the tests.